### PR TITLE
   NEPT-975: Uninstall and remove multisite_dynamic_basetheme module

### PR DESCRIPTION
**Pull request by gillesdeudon**
### Description

As part of the upgrade to version 2.3 we noticed that in the past there was a procedure to keep several ec_resp versions and choose the base theme a site wanted to use.

### Change log
- Removed:
    The multisite_dynamic_basetheme module folder.

### CODE REVIEW

**Comment by Medi4tor** [comment]
Looks OK for me. The module is removed.

File: phpcs-ruleset.xml [hunk 1]
```diff
@@ -37,6 +37,9 @@
     <!-- This module is not supported by Core team so we exclude it from QA -->
     <exclude-pattern>profiles/*/modules/custom/nexteuropa_newsroom/</exclude-pattern>
 
+    <!-- This module is not supported by Core team so we exclude it from QA, to avoid phpcs issue -->
+    <exclude-pattern>profiles/*/modules/custom/nexteuropa_newsroom/</exclude-pattern>
+
```
Comment (drupol) [inline phpcs-ruleset.xml]: I see two issues here:

1. this is out of scope
2. it's duplicated

Maybe a wrong merge ?

Comment (Medi4tor) [reply phpcs-ruleset.xml]: Nice catch Pol.
We will have to remove that.
